dhis-2/dhis-api/src/main/java/org/hisp/dhis/category/CategoryOption.java

```diff
@@ -91,6 +91,7 @@ public CategoryOption()
     public CategoryOption( String name )
     {
         this.name = name;
+        this.shortName = name;
```

Comment:
I assume this is just for testing purposes @jbee ? This is a little bit risky perhaps as we might see unintended consequences. If it is for test purposes we might want to update `DhisConvenienceTest` methods. We need to coordinate with front-end/maintenance app I assume before merging this PR.

Reply:
It is testing only but I can also update on the call side instead.

Reply:
And the `default` option uses this. And lots of callers in tests.